apps/balancer-tools/src/app/apr/round/[roundId]/page.tsx

```diff
@@ -31,6 +32,15 @@ export default function Page({
       >
         <TopPoolsChart roundId={roundId} />
       </Suspense>
+      <Suspense
+        fallback={
+          <div className="flex h-full w-full flex-col justify-center rounded-3xl">
+            <Loading />
+          </div>
```

Comment:
we've used this in so many places I wonder if `Loading` shouldn't just have that div within it 😅